packages/unstyled/radio/src/RadioIcon.tsx

```diff
@@ -3,15 +3,23 @@ import { useRadio } from './RadioProvider';
 
 export const RadioIcon = (StyledRadioIcon: any) =>
   forwardRef(({ children, ...props }: any, ref?: any) => {
-    const { isChecked } = useRadio('RadioContext');
+    const { isChecked, forceMount } = useRadio('RadioContext');
 
-    return (
-      <>
-        {isChecked && (
-          <StyledRadioIcon {...props} ref={ref}>
-            {children}
-          </StyledRadioIcon>
-        )}
-      </>
-    );
+    if (forceMount || isChecked) {
+      return (
+        <StyledRadioIcon
+          {...props}
+          ref={ref}
+          // eslint-disable-next-line react-native/no-inline-styles
+          style={{
+            ...props.style,
+            visibility: isChecked ? 'visible' : 'hidden',
```

Comment:
visibility ??

Reply:
Don't make changes in style keep this one same as previous one.

Reply:
removed style